charts/substrate-telemetry/values.yaml

```diff
@@ -32,6 +32,12 @@ envVars:
     # The frontend docker container makes this available to the UI,
     # so that it knows where to look for feed information:
     SUBSTRATE_TELEMETRY_URL: "wss://core-service-domain.com"
+    
+extraArgs:
+  shard: {}
+  core:
+    - --expose-node-ips
```

Comment:
I think we should not set `--expose-node-ips` as default here. We should respect defaults from the upstream project. Moreover this might be incompatible with older versions of the docker image.

@lexnv Was it decided to enable this as default ?

Reply:
I agree that we shouldn't have that enabled by default. Let's only expose node IPs when we need to.

Reply:
As James mentioned we should not enable this by default. It only needs enablement for W3F 1kV program

Reply:
I completely agree with not embedding `--expose-node-ips` as default here, I just wanted to show you the use case that brought me to make the modification.